frontend/app/assets/javascripts/app.js

```diff
@@ -0,0 +1,123 @@
+var myApp = angular.module('myApp', [ 'ngRoute', 'ui.bootstrap' ]);
+
+myApp.controller('MyController', [ '$scope', function($scope) {
+  $scope.currencies = [];
+
+  // put code for highchart here
+  var myChart; // global variable to access chart
+
+  $(function() {
+
+    Highcharts.setOptions({
+      global : {
+        useUTC : false
+      }
+    });
+
+    // Create the chart
+    myChart = new Highcharts.StockChart({
+      chart : {
+        renderTo : 'container'
+      },
+
+      rangeSelector : {
+        buttons : [ {
+          count : 1,
+          type : 'minute',
+          text : '1M'
+        }, {
+          count : 5,
+          type : 'minute',
+          text : '5M'
+        }, {
+          type : 'all',
+          text : 'All'
+        } ],
+        inputEnabled : false,
+        selected : 0
+      },
+
+      title : {
+        text : 'Live stock data'
+      },
+
+      exporting : {
+        enabled : false
+      },
+
+      series : (function() {
+        // generate 10 series with name "i" and empty data
+        var seriesTmp = [];
+
+        for (var i = 0; i < 10; ++i) {
+          seriesTmp.push({
+            name : i.toString(),
+            data : [ 0, 0 ]
+          });
+        }
+        return seriesTmp;
+      }())
+    });
+
+    // $('#button').click(function () {
+    // var chart = $('#container').highcharts(),
+    // series = chart.series[0];
+    // if (series.visible) {
+    // series.hide();
+    // } else {
+    // series.show();
+    // }
+    // });
+
+  });
+
+  function toggleGraph(id) {
+    var button_id = id.split("_")[1];
+    series = myChart.series[button_id];
+    if (series.visible) {
+      series.hide();
+    } else {
+      series.show();
+    }
+  }
+
+  var ws = new WebSocket("ws://localhost:9000/wsTest");
+  ws.onmessage = function(event) {
+    var quotes = JSON.parse(event.data).quotes;
+    var fill = false;
+    // initialize symbol names
+    if ($scope.currencies.length < 10) {
+      fill = true;
+    }
+
+    var x = quotes[0].timestamp;  // note: timestamps for different symbols are not necessarily equal
```

Comment:
Rather, round the timestamp up to the `second` precision. Is it enough to fix the problem?